CONTRIBUTING.md

```diff
@@ -0,0 +1,98 @@
+# Contributing
+
+## Prerequisites
+
+- `git`
+- `python >=3.8`
+- `nodejs >=18,<19`
+
+## Quick Start
+
+The command below cleans out all built artifacts, ensures a fully-configured JS/Python
+environment, and generates everything up to the documentation website and test reports.
+
+```bash
+git clone https://github.com/jupyterlite/pyodide-kernel
+cd pyodide kernel
+npm run quickstart
+```
+
+> The above installs JupyterLab, which provides the `jlpm` command, a pre-packaged
+> version of `yarn`, and is used instead of `npm` from here on out.
+
+### Serve
+
+```bash
+jlpm serve
+```
+
+> Serve the `build/` directory as `http://127.0.0.1:8000`, which contains:
```

Comment:
Maybe we can drop the quotes (`>`) for better readability?

Reply:
8f9245a , added table